icon_associations.xml

```diff
@@ -41,6 +41,14 @@
            iconType="FILE"
            pattern="(architecture|ARCHITECTURE)(\.(md|rst|txt))?$"
            icon="/icons/files/architecture.svg"/>
+    <regex fileNames="tikuiconfig.json"
+           name="Tikui"
+           iconColor="310ca3"
+           priority="1000"
+           iconType="FILE"
+           pattern="^tikuiconfig\\.json$"
+           icon="/icons/files/tikui.svg"
+    />
```

Comment:
Put it in the "T" section, alphabetically in order with the other files. And don't use a priority of 1000.